src/test/java/jalau/at18/Azul/BagTest.java

```diff
@@ -0,0 +1,36 @@
+package jalau.at18.Azul;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+import jalau.at18.azul.Bag;
+import jalau.at18.azul.MixedBag;
```

Comment:
You should not need imports from lines 10 and 11 if package is corrected with `azul` to lowercase

Reply:
With the use  lowercase package names this issue is solved